src/test_runner.cpp

```diff
@@ -80,4 +167,117 @@ int test_all_files() {
     return code;
 }
 
-int main() { return test_all_files(); }
+class Arguments {
+    std::optional<std::string_view> file = std::nullopt;
+
+  public:
```

Comment:
```suggestion
  public:
    std::vector<std::path> files;
```

если хотим возможность указывать отдельный файл, не вижу смысла ограничиваться одним файлом